friends/src/App.js

```diff
@@ -80,18 +99,22 @@ class App extends Component {
 	render() {
 		return (
 			<Wrapper>
+				<GlobalStyle />
 				<Form
 					addFriend={this.addFriend}
 					title={`You have ${
 						this.state.data.length
-					} friends. It's sad. Add more
-					friends...`}
-					buttonText={`If You Are Adding Friends This Way, You Obviously Haven't Made Them In Real Life`}
+					} friends. It's sad. Add more. ${
+						this.state.data.length > 10
```

Comment:
instead of a ternary just use `this.state.data.length > 10 && "You are gunna need a lot more"` shorthand for the if condition when you don't need the else :D